packages/mobile/src/home/WalletHome.tsx

```diff
@@ -115,6 +123,20 @@ export class WalletHome extends React.Component<Props, State> {
     // Waiting 1/2 sec before triggering to allow
     // rest of feed to load unencumbered
     setTimeout(this.tryImportContacts, 500)
+    if (tokenBalancesError || tokenBalancesLoading) {
```

Comment:
Not sure I understand this, this is only being run on `componentDidMount`, not on update? And we're showing the error even if the token fetching is loading, I see it all the time when opening the app.

Also, the banner in the top is hiding the hamburger menu, and if you dismiss it there is no way of refreshing again other than restarting the app. Did Nitya sign off on this?